packages/salesforcedx-vscode-apex-replay-debugger/src/commands/launchFromLogFile.ts

```diff
@@ -8,15 +8,15 @@
 import * as vscode from 'vscode';
 import { DebugConfigurationProvider } from '../adapter/debugConfigurationProvider';
 
-export function launchFromLogFile(editorUri: vscode.Uri) {
+export function launchFromLogFile(logFile?: string) {
```

Comment:
DebugConfigurationProvider.getConfig only needs the log file name. Change the argument from the URI to the file name and it can be used for both "SFDX: Launch Apex Replay Debugger with Current File" and "SFDX: Launch Apex Replay Debugger with Last Log File" commands.

Reply:
Done, as per requested